docs/topic-guides/hp-tuning-det/index.txt

```diff
@@ -51,6 +51,19 @@ Determined also supports other common hyperparameter search algorithms:
    configurations with ones *near* the high-performing points in the
    hyperparameter space.
 
+Handling Trial Errors and Early Stopping Requests
+-------------------------------------------------
+When a trial errors, we try to restart from the latest checkpoint unless we
+have done so ``max_restarts`` times, which is configured in the experiment
+configuration. Once we have reached ``max_restarts``, we set the trial as
+errored. For search methods that adapt to validation metric values (Adaptive
+and PBT search methods do this), we do not continue training the model even if
+the search method would typically call for us to continue training.
```

Comment:
Possibly, For search methods that adapt to validation metric values (Adaptive
and PBT search methods do this) -> For search methods that adapt to validation metric values (Adaptive
and PBT) since you already say adaptive and PBT are search methods. Also ref link to their HP page.

Reply:
okay 👍